.travis.yml

```diff
@@ -11,8 +11,9 @@ matrix:
 
 before_install:
   - docker pull postgres:latest
-  - docker run -d -p 127.0.0.1:15432:5432 postgres:latest
-  - docker ps -a
+  - docker run -d -p 15432:5432 postgres:latest
+  - docker pull mysql:5.7
+  - docker run -d -p 13306:3306 -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=test mysql:5.7
```

Comment:
Can you check one thing. I hear travis already have mysql and posgres installed and running on build already.